### DISCUSSION TITLE
yarn docs-build-api is invalid

### DISCUSSION THREAD

**Discussion by Newcandy** [react-native-elements/react-native-elements#3809] [General]
For example, Based on the content of `https://reactnativeelements.com/docs/contributing#updating-the-component-api-documentation`,  I randomly changed the DividerProps of `packages\base\src\Divider\Divider.tsx`, but running the command `yarn docs-build-api`  did not take effect, prompting Found 0 components. File `website\docs\components\Divider.mdx` didn't update. Why ?

**Comment by arpitBhalla** [comment]
Can you share what you changed?

**Reply by Newcandy** [reply]
For example, I simplified the usage of components and added a property of type.
```
import React from 'react';
import {
  View,
  StyleSheet,
  ViewProps,
  StyleProp,
  ViewStyle,
} from 'react-native';
import { RneFunctionComponent } from '../helpers';

export interface DividerProps extends ViewProps {
  /** Type of Divider  */
  type?: 'bar' | 'line';

  /**  Applies style to the divider. */
  style?: StyleProp<ViewStyle>;
}

/**  Used for interface layout to distinguish different types of content. */
export const Divider: RneFunctionComponent<DividerProps> = ({
  type = 'line',
  style,
  ...rest
}) => (
  <>
    <View
      testID="RNE__Divider"
      style={StyleSheet.flatten([
        style,
        type === 'bar' ? styles.bar : styles.line,
      ])}
      {...rest}
    />
  </>
);

const styles = StyleSheet.create({
  bar: {
    borderBottomWidth: 8,
    borderBottomColor: 'rgba(0, 0, 0, 0.03)',
  },
  line: {
    marginHorizontal: 16,
    borderBottomWidth: StyleSheet.hairlineWidth,
    borderBottomColor: 'rgba(0, 0, 0, 0.1)',
  },
});

Divider.displayName = 'Divider';
```

Adding a new component is also not ok.
1. Branch `next`
2. Run `docs-build-api` in the root directory
3. cd website && yarn start
4. The document has not changed

 I don't know where the problem is. I look forward to your reply. Thank you.

**Reply by Newcandy** [reply]
I have found the reason. Running this code on my computer returns that filePaths is an empty array, and changing the absolute path to a relative path will match it.
File `scripts\docgen\src\index.ts`:
```
const filePaths = glob.sync(path.join(rootPath, source), {
  absolute: true,
  ignore: ignoredFiles,
  onlyFiles: true,
});
```
改成
```
const filePaths = glob.sync('../../packages/' + source, {
    ignore: ignoredFiles,
    onlyFiles: true,
});
```
